src/prost/build.rs

```diff
@@ -53,6 +53,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     tonic_build::configure()
         .file_descriptor_set_path(file_descriptor_set_path.as_path())
         .compile_well_known_types(true)
+        .protoc_arg("--experimental_allow_proto3_optional")
```

Comment:
This requires protoc version >= 3.12. Shall we ensure dependency in risedev?

Reply:
For users using Ubuntu 20.04 or earlier version, it requires extra work.  😶‍🌫️ 
```
[bionic (18.04LTS)](https://packages.ubuntu.com/bionic/protobuf-compiler) (devel): compiler for protocol buffer definition files [universe]
3.0.0-9.1ubuntu1: amd64 arm64 armhf i386 ppc64el s390x
[focal (20.04LTS)](https://packages.ubuntu.com/focal/protobuf-compiler) (devel): compiler for protocol buffer definition files [universe]
3.6.1.3-2ubuntu5: amd64 arm64 armhf i386 ppc64el s390x
[jammy (22.04LTS)](https://packages.ubuntu.com/jammy/protobuf-compiler) (devel): compiler for protocol buffer definition files [universe]
3.12.4-1ubuntu7: amd64 arm64 armhf i386 ppc64el s390x
```
https://packages.ubuntu.com/search?keywords=protobuf-compiler

Reply:
I think that's inevitable 😢